docs/Argument-Processing-and-Precedence.md

```diff
@@ -211,77 +220,60 @@ Values already included in Beaker as defaults for required arguments.
-                                     :q_puppetmaster_enterpriseconsole_port         => 443,
-                                     :q_puppet_enterpriseconsole_auth_database_name => 'console_auth',
-                                     :q_puppet_enterpriseconsole_auth_database_user => 'mYu7hu3r',
-                                     :q_puppet_enterpriseconsole_database_name      => 'console',
-                                     :q_puppet_enterpriseconsole_database_user      => 'mYc0nS03u3r',
-                                     :q_database_root_password                      => '=ZYdjiP3jCwV5eo9s1MBd',
-                                     :q_database_root_user                          => 'pe-postgres',
-                                     :q_database_export_dir                         => '/tmp',
-                                     :q_puppetdb_database_name                      => 'pe-puppetdb',
-                                     :q_puppetdb_database_user                      => 'mYpdBu3r',
-                                     :q_database_port                               => 5432,
-                                     :q_puppetdb_port                               => 8081,
-                                     :q_classifier_database_user                    => 'DFGhjlkj',
-                                     :q_database_name                               => 'pe-classifier',
-                                     :q_classifier_database_password                => '~!@#$%^*-/ aZ',
-                                     :q_activity_database_user                      => 'adsfglkj',
-                                     :q_activity_database_name                      => 'pe-activity',
-                                     :q_activity_database_password                  => '~!@#$%^*-/ aZ',
-                                     :q_rbac_database_user                          => 'RbhNBklm',
-                                     :q_rbac_database_name                          => 'pe-rbac',
-                                     :q_rbac_database_password                      => '~!@#$%^*-/ aZ',
-                                     :q_install_update_server                       => 'y',
-          },
           :dot_fog                => File.join(ENV['HOME'], '.fog'),
           :ec2_yaml               => 'config/image_templates/ec2.yaml',
           :help                   => false,
-          :collect_perf_data      => false,
+          :collect_perf_data      => 'none',
+          :puppetdb_port_ssl      => 8081,
+          :puppetdb_port_nonssl   => 8080,
+          :puppetserver_port      => 8140,
+          :nodeclassifier_port    => 4433,
+          :cache_files_locally    => true,
+          :aws_keyname_modifier   => rand(10 ** 10).to_s.rjust(10,'0'), # 10 digit random number string
+          [:run_in_parallel](runner/run_in_parallel.md)        => [],
```

Comment:
Github is having issues updating... This is corrected in my last push, you can see it is correct in the files tab, but still showing up here...